.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
```

Comment:
What is this file doing?/ is it being used in the any of the worfklows?

Reply:
It's a setting file for dependabot, that is a Github-specific file that scans dependencies for vulnerabilities and suggests updates.